lib/puppet/provider/package/pip.rb

```diff
@@ -29,7 +29,11 @@ def self.parse(line)
   # that's managed by `pip` or an empty array if `pip` is not available.
   def self.instances
     packages = []
-    pip_cmd = which(cmd) or return []
+    if cmd.class == Array
```

Comment:
Why is this done? Am I missing something here? Is this in case a new provider inherits this one?